samples/interactive-tutorials/search/search_simple_query.py

```diff
@@ -15,19 +15,19 @@
 # [START retail_search_for_products_with_query_parameter]
 # Call Retail API to search for a products in a catalog using only search query.
 #
-import os
 
 from google.cloud.retail import SearchRequest, SearchServiceClient
+from utils import get_project_id
 
-project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+project_id = get_project_id()
 
 
 # get search service request:
 def get_search_request(query: str):
     default_search_placement = (
```

Comment:
Consider an f-string, e.g. f"projects/{project_id}/locations/global/catalogs/default_catalog/placements/default_search"

Reply:
It looks like this string is in several places. Also you could include it in utils.py if you're repeating the code in multiple places.

Reply:
I've opened https://github.com/googleapis/python-retail/issues/164 to track using an f-string in all samples to limit the scope of this PR

Reply:
I've opened #165 to track reducing duplicated strings

Reply:
I'm going to mark this as resolved as we're tracking the work in separate issues.